numba/tests/test_dispatcher.py

```diff
@@ -1052,6 +1052,9 @@ def foo(x):
 
         self.run_test(foo)
 
+    @unittest.skip(("Unconditionally skip. @overload does not have an error "
```

Comment:
Does this always fail? Perhaps it should be an `@xfail` so that once it starts passing, the fact it's started passing will be an error and reinstating the test won't be missed?

Reply:
Good point, will adjust, thanks.

Reply:
Done in ddd40cd.